### ISSUE TITLE
Long spawn area loading and falling into void

### ISSUE THREAD

**Issue by Volgeop** [Zrips/CMI#3478]
**Description of issue:**
When player executes /spawn and he didn't have spawn area (chunks) loaded it teleports him earlier than it starts load... so he starts falling into void. Look at the video https://streamable.com/plcs2s. Could you fix it please somehow?

---

---

**Cmi Version (using`/cmi version`):** CMI: 8.6.26.10 BungeeCord MySQL-> 8.6.26.9 

**Server Type (Spigot/Paperspigot/etc):**  Paper

**Server Version (using `/ver`):**   This server is running Paper version git-Paper-342 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)

**Comment by CloudeLecaw** [comment]
finaly the player does not die - he gets teleported to spawn when tyour server has loaded it.

What does this have to to with Jobs? you mixed 2 plugins?

**Comment by Volgeop** [comment]
Sorry, I edited it.... I really mixed it.... I created two tickets and edited the wrong one 🤣

**Comment by CloudeLecaw** [comment]
He falls to void but get back to the point where he should get teleported if the chunk is loaded

**Comment by Volgeop** [comment]
But.... why it tooks so long? It's semi-vanilla... shouldn't it firstly load it and then teleports?

**Comment by CloudeLecaw** [comment]
Yeah your right, but there is no thing you can do aggain this. Its a normal behavior if you get ported from the server. The behavior is different if you get ported to a player bcause the chunks are loaded allready then. This is what all servers does.
